### PR TITLE
Rwd

### PR DESCRIPTION
**Pull request by JanikOskar**
@pumanitro

### CODE REVIEW

File: src/pages/Home/Contact/Contact.tsx [hunk 1]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+            </Link>
+          </CheckBoxStyle>
+          <ButtonPosition>
+            <SendButton htmlType="submit">SEND</SendButton>
+          </ButtonPosition>
+        </FormStyle>
+      </Formik>
+    </ContactStyle>
+  );
+};
```
Comment (pumanitro) [inline src/pages/Home/Contact/Contact.tsx]: Can we split this file, because it seems big?

File: src/pages/Home/Contact/Contact.tsx [hunk 2]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+`}
+  ${media.md`
+  color: #F1E8E6`}
+`;
+const CheckBoxStyle = styled(FormikCheckbox)`
+  color: ${props => props.theme.colors.utils.background.mid.color};
+  ${props => props.theme.typography.body2};
+`;
+
+const SignupSchema = Yup.object().shape({
```
Comment (pumanitro) [inline src/pages/Home/Contact/Contact.tsx]: <3 Perfecto

File: src/pages/Home/Contact/Contact.tsx [hunk 3]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+      </TextUnderSectionTitle>
+      <Formik
+        initialValues={{
+          name: '',
+          email: '',
+          message: '',
+          checkBox: ``,
+        }}
+        validationSchema={SignupSchema}
+        onSubmit={() => console.log('mleko')}
```
Comment (pumanitro) [inline src/pages/Home/Contact/Contact.tsx]: Still better than dupa xd, I will add functionality here.

File: src/pages/Home/Contact/Contact.tsx [hunk 4]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+            </EmailInputText>
+          </InputBoxes>
+          <MessageText>
+            <FormikTextArea
+              name="message"
+              InputComponent={(props: any) => <MessageTextArea {...props} placeholder="Message" />}
+            />
+          </MessageText>
+          <CheckBoxStyle name="checkBox">
+            I agree to{' '}
```
Comment (pumanitro) [inline src/pages/Home/Contact/Contact.tsx]: {' '} do we need it?

File: src/pages/Home/Contact/Contact.tsx [hunk 5]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+            <Link to="/term-of-service">
+              <TermServiceAndPolicyText>term of service</TermServiceAndPolicyText>
+            </Link>{' '}
```
Comment (pumanitro) [inline src/pages/Home/Contact/Contact.tsx]: {' '} do we need it?

File: src/components/Checkbox/Checkbox.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { Checkbox as AntCheckbox } from 'antd';
+
+export const Checkbox = styled(AntCheckbox)`
+  &&& {
+    .ant-checkbox-checked .ant-checkbox-inner:after {
+      border-color: ${props => props.theme.colors.main.primary};
+    }
+    .ant-checkbox-checked .ant-checkbox-inner {
+      color: black;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/Checkbox/Checkbox.tsx]: deleted, done.

File: src/components/Checkbox/Checkbox.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+      background-color: ${props => props.theme.colors.utils.background.mid.color};
+    }
+    .ant-checkbox-inner {
+      border: 1px solid ${props => props.theme.colors.utils.border.light};
+      width: 20px;
+      height: 20px;
+      box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.02), 0px 1px 3px rgba(50, 50, 93, 0.15);
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/Checkbox/Checkbox.tsx]: We dont need him, done

File: src/components/FormikFields/FormikTextArea/FormikTextArea.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+type FormikTextAreaProps = InterfaceInputComponent & MemoFormikInputProps;
+
+const CustomInputComponent = (props: FormikTextAreaProps) => {
+  const { field, form, value, isValid = true, InputComponent, ...fieldProps } = props;
+  const inputValue = value || get(form.values, field.name);
+
+  const StyledTextArea = InputComponent || AntdInput;
+
+  return (
+    // @ts-ignore
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/FormikFields/FormikTextArea/FormikTextArea.tsx]: JSX element type 'StyledTextArea' does not have any construct or call signatures. That's why

File: src/components/FormikFields/FormikTextArea/FormikTextArea.tsx [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+      value={inputValue}
+      onChange={(event: ChangeEvent<HTMLInputElement>) => {
+        setFieldValueAndTouched(field.name, event.target.value, form);
+      }}
+      {...fieldProps}
+    />
+  );
+};
+
+// eslint-disable-next-line react/display-name
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/FormikFields/FormikTextArea/FormikTextArea.tsx]: We dont need it

File: src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx [hunk 10]
```diff
@@ -73,43 +81,76 @@ const TermOfServiceText = styled.span`
...
+  const { more } = useRWD();
+  return more.md ? (
+    <FooterBG>
+      <div>
+        <AboutUs>About us</AboutUs>
+        <ContactAddress>
+          Zbożowa 4A 70-653 Szczecin Poland
+          <br />
+          +48 783 697 219 <br />
+          patrykjanik1710@gmail.com
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx]: Done.

File: src/components/FormikFields/FormikTextArea/FormikTextArea.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/FormikFields/FormikTextArea/FormikTextArea.tsx]: Done.

File: src/global/Layouts/HeaderPageFooter/Header/Header.tsx [hunk 11]
```diff
@@ -4,6 +4,8 @@ import LogoSvg from 'assets/logo.svg';
 import { Link as LinkRouter } from 'react-router-dom';
 import Headroom from 'react-headroom';
 import { scrollTo } from '../HeaderFooterCommon';
+import { HamburgerMenu } from './HamburgerMenu/HamburgerMenu';
+import { media, useRWD } from '../../../RWD';
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Header/Header.tsx]: Done.

File: src/global/Layouts/HeaderPageFooter/Header/Header.tsx [hunk 12]
```diff
@@ -42,26 +50,37 @@ const HeaderText = styled.h6`
...
+      <HeaderWrapper>
+        {less.md && <HamburgerMenu />}
+        <LinkRouter to="/">
+          <LogoSvgStyle src={LogoSvg} alt="" />
         </LinkRouter>
-      </LinksPosition>
-    </HeaderWrapper>
-  </HeaderScroll>
-);
+        {more.md && (
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Header/Header.tsx]: Done. :D

File: src/pages/Home/Banner/Banner.tsx [hunk 13]
```diff
@@ -14,49 +16,121 @@ const BannerStyle = styled.div`
...
+      flex-direction: row;
+  `}
 `;
+
 const GroupedTextWithButton = styled.div`
   margin-right: 40px;
+  display: flex;
+  ${media.xs`
+  align-items:center;
+  margin:0 0 0 0 ;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: Done.

File: src/pages/Home/Banner/Banner.tsx [hunk 14]
```diff
@@ -14,49 +16,121 @@ const BannerStyle = styled.div`
...
   color: ${props => props.theme.colors.utils.background.mid.color};
+  ${props => props.theme.typography.body1};
+  white-space: pre-wrap;
   margin-bottom: 40px;
-  line-height: 19px;
-  z-index: ${props => props.theme.typography.body1};
+  ${media.xs`
+    width:100%;
+  `}
+  ${media.md`
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: Done. We dont need media.md

File: src/pages/Home/Banner/Banner.tsx [hunk 15]
```diff
@@ -14,49 +16,121 @@ const BannerStyle = styled.div`
...
+`}
+  ${media.md`
+   width: 199px;
+  height: 376px;
+  left: -0px;
+  top: 480px;
+  `}
+`;
+
+const BannerPhotoImg = styled.img``;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: Nope.

File: src/pages/Home/Contact/Contact.tsx [hunk 16]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+  ${media.md`
   padding: 96px 192px 80px 192px;
+  `}
 `;
 
 const TextUnderSectionTitle = styled.h5`
   color: ${props => props.theme.colors.utils.background.mid.color};
-  margin: 24px 0 92px 16px;
+  ${media.xs`
+  margin: 24px 0 0 0  ;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Done.

File: src/pages/Home/Contact/Contact.tsx [hunk 17]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+  margin: 24px 0 96px 16px;
+  `}
 `;
 
 const InputBoxes = styled.div`
-  display: flex;
-  justify-content: space-between;
-  margin-left: 16px;
+  margin: 0 0 32px 0;
+  ${media.xs`
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Nope.Done

File: src/pages/Home/Contact/Contact.tsx [hunk 18]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+const TermServiceAndPolicyText = styled.span`
+  ${props => props.theme.typography.body2};
+  cursor: pointer;
+  color: ${props => props.theme.colors.utils.background.mid.color};
+  &:hover {
+    color: ${props => props.theme.colors.main.secondary};
+    text-decoration-line: ${props => props.theme.textDecorationLine};
+  }
+  ${media.xs`
+   color: #F55951;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Yes, but we cant use they like ${props => props.theme ... in ${media} an error pops up

File: src/pages/Home/Contact/Contact.tsx [hunk 19]
```diff
@@ -3,87 +3,225 @@ import styled from 'styled-components';
...
+`}
+  ${media.md`
+  color: #F1E8E6`}
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Yes, but we cant use they like ${props => props.theme ... in ${media} an error pops up

File: src/pages/Home/TheProcess/TheProcess.tsx [hunk 20]
```diff
@@ -9,85 +9,169 @@ import RequirementsIcon from 'assets/RequirementsIcon.svg';
...
+  right: 0;
+  bottom: -1800px;
+  `}
+  ${media.md`
   width: 77px;
   height: 376px;
   right: 0px;
-  bottom: -1620px;
-  background-color: ${props => props.theme.colors.main.secondary};
+  bottom: -1280px;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Cuz its Icon and we need set backgroudn-color or we have black as primary

File: src/pages/Home/TheProcess/TheProcess.tsx [hunk 21]
```diff
@@ -9,85 +9,169 @@ import RequirementsIcon from 'assets/RequirementsIcon.svg';
...
 `;
 
 const Shape = styled(Icon)`
   position: absolute;
+  background-color: ${props => props.theme.colors.main.secondary};
+  ${media.xs`
+  width:27px;
+  height:210px;
+  right: 0;
+  bottom: -1800px;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: What? @pumanitro

File: src/pages/Home/TheProcess/TheProcess.tsx [hunk 22]
```diff
@@ -9,85 +9,169 @@ import RequirementsIcon from 'assets/RequirementsIcon.svg';
...
+
 const ProcessTimeLine = styled.div`
   margin: 40px 96px 60px 0;
   flex-direction: column;
   display: flex;
   align-items: center;
+  ${media.xs`
+    display: flex;
+  align-items: center;
+  margin: 32px 0 0 0;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Done.

File: src/pages/Home/Contact/Contact.tsx (hunk 2 above)
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Done.

File: src/pages/Home/Contact/Contact.tsx (hunk 3 above)
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Done xD

File: src/components/FormikFields/FormikTextArea/FormikTextArea.tsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/FormikFields/FormikTextArea/FormikTextArea.tsx]: @pumanitro

File: src/pages/TermOfService/TermOfService.tsx [hunk 23]
```diff
@@ -8,14 +9,25 @@ export interface TermServiceProps {
 
 const MainComponent = styled.div`
   background-color: ${props => props.theme.colors.main.primary};
-  padding: 40px 80px 40px 80px;
   ${props => props.theme.typography.body1};
   color: ${props => props.theme.colors.utils.background.mid.color};
+  ${media.xs`
+    padding: 20px 20px 20px 20px;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/TermOfService/TermOfService.tsx]: Done.

File: src/pages/TermOfService/TermOfService.tsx [hunk 24]
```diff
@@ -8,14 +9,25 @@ export interface TermServiceProps {
...
+    white-space: pre-wrap;
+  `}
+  ${media.md`
+  padding: 40px 80px 40px 80px;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/TermOfService/TermOfService.tsx]: Done.

File: src/pages/TermOfService/TermOfService.tsx [hunk 25]
```diff
@@ -8,14 +9,25 @@ export interface TermServiceProps {
...
 `;
 
 const TextH4 = styled.h4`
   color: ${props => props.theme.colors.utils.background.mid.color};
   margin: 20px 0 20px 0;
+  ${media.xs`
+    font-size: 24px;
+    line-height:24px;
+    `}
+  ${media.md``}
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/TermOfService/TermOfService.tsx]: Done.

File: src/pages/TermOfService/TermOfService.tsx [hunk 26]
```diff
@@ -1,5 +1,6 @@
 import React, { FunctionComponent } from 'react';
 import styled from 'styled-components';
+import { media } from '../../global/RWD';
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/TermOfService/TermOfService.tsx]: Done.

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 20 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: @pumanitro

File: src/pages/Home/Contact/Contact.tsx (hunk 1 above)
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: @pumanitro  we can but as if you saw it?

File: src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx [hunk 27]
```diff
@@ -73,43 +81,76 @@ const TermOfServiceText = styled.span`
...
-    <LeftShape svgLink={Shape4} />
-    <RightShape svgLink={Shape5} />
-  </FooterBG>
-);
+      </BoxWithLinks>
+      <LeftShape svgLink={Shape4} />
+      <RightShape svgLink={Shape5} />
+    </FooterBG>
+  ) : (
+    <FooterBG>
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx]: Done.

File: src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx [hunk 28]
```diff
@@ -0,0 +1,113 @@
...
+
+  return (
+    <div>
+      <HamburgerMenuIcon color={themeContext.colors.utils.background.light} onClick={() => setIsMenuOpened(true)} />
+      <StyledDrawer placement="left" closable={false} onClose={() => setIsMenuOpened(false)} visible={isMenuOpened}>
+        <MenuDivider />
+        <DrawerWrapper>
+          <StyledLink
+            onClick={() => {
+              scrollTo('Process');
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx]: Done.

File: src/pages/Home/Banner/Banner.tsx [hunk 29]
```diff
@@ -14,49 +16,121 @@ const BannerStyle = styled.div`
...
+        </SecondText>
+        <Link to="Contact" spy={true} smooth={true} duration={500}>
+          <ContactButton type="secondary">Contact Us</ContactButton>
+        </Link>
+        <Shape svgLink={Shape1} />
+      </GroupedTextWithButton>
+      <BannerPhotoImg src={BannerPhoto} alt="" />
+    </BannerStyle>
+  ) : (
+    <BannerStyle>
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: Done.

File: src/pages/Home/Contact/Contact.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: @pumanitro

File: src/pages/PrivatePolicy/PrivatePolicy.tsx [hunk 30]
```diff
@@ -14,10 +15,16 @@ export interface PolicyProps {
 
 const PrivacyPolicyBg = styled.div`
   background-color: ${props => props.theme.colors.main.primary};
-  padding: 36px 80px 48px 80px;
   display: flex;
   justify-content: center;
   align-items: center;
+  ${media.xs`
+    padding: 20px 20px 20px 20px;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/PrivatePolicy/PrivatePolicy.tsx]: Done.

File: src/pages/PrivatePolicy/PrivatePolicy.tsx [hunk 31]
```diff
@@ -1,5 +1,6 @@
 import React, { FunctionComponent } from 'react';
 import styled from 'styled-components';
+import { media } from '../../global/RWD';
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/PrivatePolicy/PrivatePolicy.tsx]: Done.

File: src/pages/Home/TheProcess/TimeLineElement/TimeLineElement.tsx [hunk 32]
```diff
@@ -70,5 +95,16 @@ export const TimeLineElement: FunctionComponent<ElementProps> = (props: ElementP
       </ProcessTimeLineElement>
       {isWithLine && <LineUnderSvg svgLink={Line} />}
     </>
+  ) : (
+    <>
+      <ProcessTimeLineElement>
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TimeLineElement/TimeLineElement.tsx]: Done.

File: src/pages/Home/TheProcess/TheProcess.tsx [hunk 33]
```diff
@@ -9,85 +9,169 @@ import RequirementsIcon from 'assets/RequirementsIcon.svg';
...
+            isWithLine={false}
+          />
+        </ProcessTimeLine>
+        <BottomText>*Designs can be provided </BottomText>
+      </ProcessFrame>
+      <Shape svgLink={Shape2} />
+    </>
+  ) : (
+    <>
+      <ProcessFrame className="Process" id="Process">
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Done.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx [hunk 34]
```diff
@@ -0,0 +1,162 @@
...
+          <QuotationBoxWithText>{TestimonialText}</QuotationBoxWithText>
+          <BoxWithSecondQuote>
+            <QuoteSign svgLink={Quote} />
+          </BoxWithSecondQuote>
+          <AuthorText>{TestimonialPerson}</AuthorText>
+        </QuotationBoxWithQuoteSvg>
+      </PortfolioDescription>
+    </CarouselWhole>
+  ) : (
+    <CarouselWhole>
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Done.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx [hunk 35]
```diff
@@ -0,0 +1,162 @@
...
+const CarouselWhole = styled.div`
+  display: flex !important;
+  justify-content: center;
+  ${media.xs`
+  flex-direction:column;
+  align-items:center;
+  `}
+  ${media.md`
+  flex-direction:row;
+   align-items: center;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Done.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx [hunk 36]
```diff
@@ -0,0 +1,162 @@
+import React, { FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { Icon } from 'components/Icon/Icon';
+import Quote from 'assets/Quote.svg';
+import { media, useRWD } from '../../../global/RWD';
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Done.

File: src/pages/Home/Contact/Contact.tsx (hunk 4 above)
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Yes, cuz we need space between this words

File: src/pages/Home/Contact/Contact.tsx (hunk 5 above)
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Yes, cuz we need space between this words

File: src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx]: I can't see this change. Please double-check @JanikOskar

File: src/pages/Home/Contact/Contact.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/pages/Home/Contact/Contact.tsx]: @JanikOskar We need to find a way to use there props. But we can omit it this time.

File: src/pages/Home/Contact/Contact.tsx [hunk 37]
```diff
@@ -3,87 +3,224 @@ import styled from 'styled-components';
...
+            </EmailInputText>
+          </InputBoxes>
+          <MessageText>
+            <FormikTextArea
+              name="message"
+              InputComponent={(props: any) => <MessageTextArea {...props} placeholder="Message" />}
+            />
+          </MessageText>
+          <CheckBoxStyle name="checkBox">
+            I agree to{' '}
```
Comment (pumanitro) [inline src/pages/Home/Contact/Contact.tsx]: image
Checkbox and links looks differently than on figma mockups.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx (hunk 36 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Are you sure? I can't see a change.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx (hunk 35 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Are you sure? I can't see a change.

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 21 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/pages/Home/TheProcess/TheProcess.tsx]: 54 line. @JanikOskar

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 20 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/pages/Home/TheProcess/TheProcess.tsx]: @JanikOskar I'm talking about 60 line, it is this negative huge bottom.

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 33 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Are you sure? I can't see a change.

File: src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx [hunk 38]
```diff
@@ -0,0 +1,92 @@
...
+import HamburgerMenuSvg from 'assets/hamburger-menu.svg';
+import { Icon } from 'components/Icon/Icon';
+import { css, ThemeContext } from 'styled-components';
+import { Drawer } from 'antd';
+import styled from 'styled-components';
+import { LinkWithoutIsActiveInDom, StyledLinkProps } from '../common';
+import { scrollTo } from '../../HeaderFooterCommon';
+
+const LinkStyles = css<StyledLinkProps>`
+  height: 50px;
```
Comment (pumanitro) [inline src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx]: image
On figma w have different color of hamburger menu.

File: src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx [hunk 39]
```diff
@@ -0,0 +1,92 @@
...
+
+export const HamburgerMenu = () => {
+  const themeContext = useContext(ThemeContext);
+  const [isMenuOpened, setIsMenuOpened] = useState(false);
+  const HamburgerFunction = (place: string) => {
+    scrollTo(place);
+    setIsMenuOpened(false);
+  };
+  return (
+    <div>
```
Comment (pumanitro) [inline src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx]: image
I have added designs for hamburger menu. Please adjust styles to it and add icons.

File: src/pages/Home/TheProcess/TheProcess.tsx [hunk 40]
```diff
@@ -9,85 +9,167 @@ import RequirementsIcon from 'assets/RequirementsIcon.svg';
...
-          title="Idea"
-          content="Everything starts with your idea"
-          contentPosition="left"
-        />
+export const TheProcess: FunctionComponent = () => {
+  const { more } = useRWD();
+  return more.md ? (
+    <>
+      <ProcessFrame className="Process" id="Process">
+        <SectionTitle section="The process" color="primary" boxColor="secondary" />
```
Comment (pumanitro) [inline src/pages/Home/TheProcess/TheProcess.tsx]: image
- To much space here
- adjust dashed line to be as on figma
- please remove "enters" on mobile

File: src/pages/Home/OurService/OurService.tsx [hunk 41]
```diff
@@ -10,72 +10,127 @@ import App from 'assets/App.svg';
...
+            We specialize in web technologies -<br /> simply saying we are in love with
+            <br /> <RedText>JavaScript!</RedText>
+            <br />
+            <br />
+            <br />
+            Those are few we know at expert level:
+          </WholeTextPre>
+        )}
+      </OurSevicesBox>
+      <TechnicalsRow>
```
Comment (pumanitro) [inline src/pages/Home/OurService/OurService.tsx]: image
- fix margins like on figma
- fix this line on pwa image

File: src/pages/Home/OurService/OurService.tsx [hunk 42]
```diff
@@ -10,72 +10,127 @@ import App from 'assets/App.svg';
...
+        <IconsSvgs src={JsTs} alt="" />
```
Comment (pumanitro) [inline src/pages/Home/OurService/OurService.tsx]: image
- to small margins (top, bottom, phone PC)
- this shape shouldn't be here.

File: src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx]: Done.

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 33 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Done.

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 20 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Done.

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 21 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Done.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx (hunk 35 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Done.

File: src/pages/Home/Portfolio/PortfolioProperties.tsx (hunk 36 above)
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Portfolio/PortfolioProperties.tsx]: Yep. Done

File: src/pages/Home/Banner/Banner.tsx [hunk 43]
```diff
@@ -5,6 +5,8 @@ import { Button } from 'components/Button/Button';
 import Shape1 from 'assets/Shape1.svg';
 import { Icon } from 'components/Icon/Icon';
 import { Link } from 'react-scroll';
+import { media, useRWD } from 'global/RWD';
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: DOne.

File: src/pages/Home/Contact/Contact.tsx (hunk 37 above)
Comment (JanikOskar) [reply src/pages/Home/Contact/Contact.tsx]: Done

File: src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx (hunk 38 above)
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx]: Done

File: src/pages/Home/TheProcess/TheProcess.tsx (hunk 40 above)
Comment (JanikOskar) [reply src/pages/Home/TheProcess/TheProcess.tsx]: Done.

File: src/pages/Home/OurService/OurService.tsx (hunk 41 above)
Comment (JanikOskar) [reply src/pages/Home/OurService/OurService.tsx]: DONE.

File: src/pages/Home/OurService/OurService.tsx (hunk 42 above)
Comment (JanikOskar) [reply src/pages/Home/OurService/OurService.tsx]: Done.

File: src/pages/Home/Banner/Banner.tsx [hunk 44]
```diff
@@ -5,58 +5,119 @@ import { Button } from 'components/Button/Button';
...
-  </BannerStyle>
-);
+export const Banner: FunctionComponent = () => {
+  const { less, more } = useRWD();
+  return (
+    <MaxWidthWithBg BgColor="primary">
+      <Row>
+        <BannerStyle>
+          <GroupedTextWithButton>
+            <Col lg={24} xl={20} xxl={20}>
```
Comment (pumanitro) [inline src/pages/Home/Banner/Banner.tsx]: one xl is enough

File: src/pages/Home/Banner/Banner.tsx [hunk 45]
```diff
@@ -5,58 +5,119 @@ import { Button } from 'components/Button/Button';
...
+              <SecondText>
+                We are here to help you succeed. That is why we provide our time and top quality code. We care and take
+                responsibility for your products - you can focus on your business strategies!
+              </SecondText>
+            </Col>
+            <Link to="Contact" spy={true} smooth={true} duration={500}>
+              <ContactButton type="secondary">Contact Us</ContactButton>
+            </Link>
+          </GroupedTextWithButton>
+          <Col lg={8} xl={8} xxl={8}>
```
Comment (pumanitro) [inline src/pages/Home/Banner/Banner.tsx]: one 8 size is enough

File: src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (pumanitro) [reply src/global/Layouts/HeaderPageFooter/Footer/Footer.tsx]: NOPE.

File: src/components/Checkbox/Checkbox.tsx [hunk 46]
```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { Checkbox as AntCheckbox } from 'antd';
+
+export const Checkbox = styled(AntCheckbox)`
+  &&& {
+    yarn .ant-checkbox-checked .ant-checkbox-inner:after {
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/Checkbox/Checkbox.tsx]: nope.done

File: src/components/MaxSizeAndBackground/MaxWidthAndBg.tsx [hunk 47]
```diff
@@ -0,0 +1,31 @@
...
+  children?: React.ReactNode;
+}
+
+const MaxWidth = styled.div`
+  max-width: 1440px;
+  margin: 0 auto;
+  float: none;
+`;
+
+type BgProps = { BgColor?: ColorBacground };
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/components/MaxSizeAndBackground/MaxWidthAndBg.tsx]: done.change

File: src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx [hunk 48]
```diff
@@ -0,0 +1,118 @@
...
+const HamburgerIcon = styled(Icon)`
+  .active {
+    background-color: red !important;
+  }
+`;
+
+const TextInHamburger = styled.div`
+  margin-left: 20px;
+  .active {
+    color: red !important;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx]: change. done.

File: src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx [hunk 49]
```diff
@@ -0,0 +1,118 @@
...
+  display: flex;
+  align-items: center;
+  padding-left: 40px;
+  cursor: pointer;
+  &.active {
+  background-color: ${props => props.theme.colors.main.tetiary};
+  }
+  &.active {  
+    ${HamburgerIcon} {
+    background-color: ${props => props.theme.colors.main.secondary} !important;
```
[the remark this answers is not in the document]
Comment (JanikOskar) [reply src/global/Layouts/HeaderPageFooter/Header/HamburgerMenu/HamburgerMenu.tsx]: done. yes

File: src/pages/Home/Banner/Banner.tsx (hunk 44 above)
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: change done

File: src/pages/Home/Banner/Banner.tsx (hunk 45 above)
Comment (JanikOskar) [reply src/pages/Home/Banner/Banner.tsx]: change done.

**Pull request merged**